### PR TITLE
small fixes/Latex code cleanup

### PR DESCRIPTION
**Pull request by entorb**
Minor "invisible" Latex code cleanup: Paragraphs in one line, fixing spaces, dropping indentation

fixing a few quotations and "\emph in \emph"

### CODE REVIEW

File: chapters/hpmor-chapter-007.tex [hunk 1]
```diff
@@ -452,7 +452,7 @@ \chapter{Reciprocation}
 
 It was at this point that Harry came to the sudden realisation that (a)~the sounds coming from the rest of the train platform had turned into more of a blurred white noise at around the same time Draco had reached inside his robes, and (b)~when he had discussed committing murder as a bonding method, there had been exactly one person in the conversation who’d thought they were joking.
 
-\emph{Right. Because he} seemed \emph{like such a normal kid. And he} is \emph{a normal kid, he is just what you’d} expect \emph{a baseline male child to be like if Darth Vader were his doting father.}
+\emph{Right. Because he \emph{seemed} like such a normal kid. And he} is \emph{a normal kid, he is just what you’d \emph{expect} a baseline male child to be like if Darth Vader were his doting father.}
```
Comment (rrthomas) [inline chapters/hpmor-chapter-007.tex:455]: One more you missed on "is" in the first line!

File: chapters/hpmor-chapter-013.tex [hunk 2]
```diff
@@ -160,10 +160,8 @@ \chapter{Asking the Wrong Questions}
 “How can I earn more points?” Harry said out loud.
 
 Then he pulled out the drawer.
-\begin{align*}\intertext{\scshape \centering
-Opportunities to do good are everywhere\\
-but darkness is where the light needs to be}
-\hbox{\scshape Cost of question: }&\hbox{\scshape 1 point}\\
+\begin{align*}\intertext{\scshape \centering Opportunities to do good are everywhere\\
```
Comment (rrthomas) [inline chapters/hpmor-chapter-013.tex:163]: Please undo this one, the markup's much easier to read with the newlines, and it also gives a better impression of the text.

File: chapters/hpmor-chapter-015.tex [hunk 3]
```diff
@@ -62,15 +62,15 @@ \chapter{Conscientiousness}
 Professor McGonagall stood up and moved over to the wall behind her desk, which held a polished wooden board. “There are many reasons why Transfiguration is dangerous, but one reason stands above all the rest.” She produced a short quill with a thick end and used it to sketch letters in red, which she then underlined, using the same marker, in blue:
 
 \begin{center}
-  \newsavebox{\hpbox}%
```
Comment (rrthomas) [inline chapters/hpmor-chapter-015.tex:65]: Please don't remove this indentation.

File: chapters/hpmor-chapter-063.tex [hunk 4]
```diff
@@ -887,7 +887,7 @@ \section{Aftermath, Hermione Granger:}
...
+\emph{I will say this much, Mr~Potter: you are already an Occlumens, and I think you will become a perfect Occlumens before long. Identity does not mean, to such as us, what it means to other people. Anyone we can imagine, we can be; and the true difference about you, Mr~Potter, is that you have an unusually good imagination. A playwright must contain his characters, he must be larger than them in order to enact them within his mind. To an actor or spy or politician, the limit of his own diameter is the limit of who he can pretend to be, the limit of which face he may wear as a mask. But for such as you and I, anyone we can imagine, we can \emph{be}, in reality and not pretence. While you imagined yourself a child, Mr~Potter, you \emph{were} a child. Yet there are other existences you could support, larger existences, if you wished. Why are you so free, and so great in your circumference, when other children your age are small and constrained? Why can you imagine and \emph{become}selves more adult than a mere child of a playwright should be able to compose? That I do not know, and I must not say what I guess. But what you have, Mr~Potter, is freedom.}
```
Comment (rrthomas) [inline chapters/hpmor-chapter-063.tex:890]: missing space after `\emph{become}`.

File: chapters/hpmor-chapter-064.tex [hunk 5]
```diff
@@ -10,14 +10,13 @@
 \makeatother
 
 \newcommand{\OmakeIVsection}[2][1.5]{%
-  \OmakeIVspecialsection[#1]{\MakeUppercase{#2}}}
```
Comment (rrthomas) [inline chapters/hpmor-chapter-064.tex:13]: Please restore this indentation.

**Comment by rrthomas** [comment]
Good stuff, I've made some comments.

File: chapters/hpmor-chapter-007.tex (hunk 1 above)
Comment (entorb) [reply chapters/hpmor-chapter-007.tex:455]: thanks, good catch!

File: chapters/hpmor-chapter-013.tex (hunk 2 above)
Comment (entorb) [reply chapters/hpmor-chapter-013.tex:163]: good catch, fixed

File: chapters/hpmor-chapter-063.tex (hunk 4 above)
Comment (entorb) [reply chapters/hpmor-chapter-063.tex:890]: ups, sorry, fixed

File: chapters/hpmor-chapter-015.tex [hunk 6]
```diff
@@ -62,15 +62,15 @@ \chapter{Conscientiousness}
...
-\newsavebox{\hpbox}%
-\fontspec[ExternalLocation,Color=AA0000]{Florante}
-\savebox{\hpbox}{\MakeUppercase{Transfiguration is not permanent!}}
-\vspace{0.5ex}
-\usebox{\hpbox}
-\settowidth{\versewidth}{\usebox{\hpbox}}
-\vskip -1ex
-\fontspec[ExternalLocation,Color=2020FF]{ArchitectsDaughter}
-\resizebox{\versewidth}{.6ex}{\rotatebox{90}{I}}
+  \newsavebox{\hpbox}%
```
Comment (rrthomas) [inline chapters/hpmor-chapter-015.tex:65]: This change of indentation looks like it is meant for a different PR.

File: chapters/hpmor-chapter-064.tex [hunk 7]
```diff
@@ -10,13 +10,14 @@
 \makeatother
 
 \newcommand{\OmakeIVsection}[2][1.5]{%
-\OmakeIVspecialsection[#1]{\MakeUppercase{#2}}}
+  \OmakeIVspecialsection[#1]{\MakeUppercase{#2}}}
```
Comment (rrthomas) [inline chapters/hpmor-chapter-064.tex:13]: For PR #66?

File: chapters/hpmor-chapter-064.tex [hunk 8]
```diff
@@ -10,13 +10,14 @@
...
 
 \chapter{Omake Files IV, Alternate Parallels}
 
 \OmakeIVspecialsection[1.6]{\fontspec[ExternalLocation]{RingBearer}
-\settowidth{\versewidth}{\mbox{the}} Lord\scalebox{.40}{\parbox[b]{\versewidth}{%
-\centering of\\\nointerlineskip\vskip 4pt the}}Ratîonalît\raisebox{-.32ex}{Y}}
+\settowidth{\versewidth}{\mbox{the}}
+Lord\scalebox{.40}{\parbox[b]{\versewidth}{%
+        \centering of\\\nointerlineskip\vskip 4pt the}}Ratîonalît\raisebox{-.32ex}{Y}}
```
Comment (rrthomas) [inline chapters/hpmor-chapter-064.tex:20]: For PR #66?

File: chapters/hpmor-chapter-064.tex (hunk 7 above)
Comment (entorb) [reply chapters/hpmor-chapter-064.tex]: good point, looks I modified it in both, now copied there too

File: chapters/hpmor-chapter-064.tex (hunk 8 above)
Comment (entorb) [reply chapters/hpmor-chapter-064.tex:19]: good point, looks I modified it in both, now copied there too

**Comment by rrthomas** [comment]
One more comment you didn't reply to, then I guess rebase and see how it looks. I am guessing there are conflicts with master now as `Rebase and merge` is greyed out.

File: chapters/hpmor-chapter-015.tex (hunk 6 above)
Comment (entorb) [reply chapters/hpmor-chapter-015.tex]: sorry, forgot to reply, now as desired in master and no change of that file in this PR

**Comment by rrthomas** [comment]
OK, now please can you rebase this on master? (That is, without the merge of master into this branch.)

**Comment by entorb** [comment]
Gosh, I guess I did it the wrong way round: I did "Merge branch 'master' into small-fixes”. Guess that is why I can not squash?

**Comment by rrthomas** [comment]
Possibly easiest to `git reset` your branch to just before the merge, then rebase.

**Comment by entorb** [comment]
thanks, should have worked, thanks for the support!

File: chapters/hpmor-chapter-013.tex [hunk 9]
```diff
@@ -171,8 +171,7 @@ \chapter{Asking the Wrong Questions}
 Then he pulled out the drawer.
 \begin{align*}\intertext{\scshape \centering
 Opportunities to do good are everywhere\\
-but darkness is where the light needs to be}
-\hbox{\scshape Cost of question: }&\hbox{\scshape 1 point}\\
+but darkness is where the light needs to be} \hbox{\scshape Cost of question: }&\hbox{\scshape 1 point}\\
```
Comment (rrthomas) [inline chapters/hpmor-chapter-013.tex:174]: Please revert this, I think the markup is clearer in the original.

Comment (entorb) [reply chapters/hpmor-chapter-013.tex]: done

File: chapters/hpmor-chapter-012.tex [hunk 10]
```diff
@@ -200,8 +200,7 @@ \chapter{Impulse Control}
 
 But he \emph{could,} there was no rule saying he couldn’t, he \emph{could} just pretend he’d never heard that little whisper. Let the universe go on in exactly the same way it would have if that one critical moment had never occurred. Twenty years later, that was what he would desperately wish had happened twenty years ago, and twenty years before twenty years later happened to be right now. Altering the distant past was easy, you just had to think of it at the right time.
 
-Or…this was even \emph{more} counterintuitive…he could even
-inform, oh, say, \emph{Professor McGonagall,} instead of Draco \emph{or} Hermione. And she could get a few good people together and get that little extra spell taken off the Hat.
+Or…this was even \emph{more} counterintuitive…he could even inform, oh, say, \emph{Professor McGonagall,} instead of Draco \emph{or} Hermione. And she could get a few good people together and get that little extra spell taken off the Hat.
```
Comment (rrthomas) [inline chapters/hpmor-chapter-012.tex:203]: Should the comma be outside the `\emph` in `\emph{Professor McGonagall,}`?

File: chapters/hpmor-chapter-038.tex [hunk 11]
```diff
@@ -10,8 +10,7 @@ \chapter{The Cardinal Sin}
 
 And finally, Harry drew his wand; and he couldn’t help thinking of the parents he’d only just kissed goodbye, of the world whose problems he was leaving behind…
 
-With a strange feeling of guilt for the unavoidable, Harry said,
-“\emph{Thermos.}”
+With a strange feeling of guilt for the unavoidable, Harry said, “\emph{Thermos.}”
```
Comment (rrthomas) [inline chapters/hpmor-chapter-038.tex:13]: What about here, should the full stop be outside the `\emph`?

File: chapters/hpmor-chapter-012.tex (hunk 10 above)
Comment (entorb) [reply chapters/hpmor-chapter-012.tex:203]: done, but I assume there will be many more. I will try to include such a check in check-chapters.py / https://github.com/rrthomas/hpmor/pull/69

File: chapters/hpmor-chapter-000.tex [hunk 12]
```diff
@@ -34,9 +34,9 @@ \section*{Something, somewhere, somewhen, must have happened differently…}
 % \end{center}
 
 \section*{Some notes}
-The opinions of characters in this story are not necessarily those of the author. What warm!Harry thinks is \emph{often} meant as a good pattern to follow, especially if Harry thinks about how he can cite scientific studies to back up a particular principle. But not everything Harry does or thinks is a good idea. That wouldn't work as a story. And the less warm characters may sometimes have valuable lessons to offer, but those lessons may also be dangerously double-edged.
+The opinions of characters in this story are not necessarily those of the author. What warm!Harry thinks is \emph{often} meant as a good pattern to follow, especially if Harry thinks about how he can cite scientific studies to back up a particular principle. But not everything Harry does or thinks is a good idea. That wouldn’s work as a story. And the less warm characters may sometimes have valuable lessons to offer, but those lessons may also be dangerously double-edged.
```
Comment (rrthomas) [inline chapters/hpmor-chapter-000.tex:37]: "wouldn't" got changed to "wouldn’s"!

File: chapters/hpmor-chapter-000.tex [hunk 13]
```diff
@@ -34,9 +34,9 @@ \section*{Something, somewhere, somewhen, must have happened differently…}
...
 
-If you haven't visited \url{https://hpmor.com}, don't forget to do that at some point; otherwise you'll miss out on the fan art, how to learn everything Harry knows, and more.
+If you haven‘t visited \url{https://hpmor.com}, don’t forget to do that at some point; otherwise you'll miss out on the fan art, how to learn everything Harry knows, and more.
```
Comment (rrthomas) [inline chapters/hpmor-chapter-000.tex:39]: Wrong direction of curly quote in the change!

File: chapters/hpmor-chapter-002.tex [hunk 14]
```diff
@@ -29,7 +29,7 @@ \chapter{Everything I Believe Is False}
 
 His father was lowered carefully to the ground.
 
-Harry ruffled a hand through his own hair. Maybe it was just that strange part of him which had \emph{already} been convinced, but…“That’s a bit of an anticlimax,” Harry said. “You’d think there’d be some kind of more dramatic mental event associated with updating on an observation of infinitesimal probability—” Harry stopped himself. Mum, the witch, and even his Dad were giving him \emph{that look} again. “I mean, with finding out that everything I believe is false.”
+Harry ruffled a hand through his own hair. Maybe it was just that strange part of him which had \emph{already} been convinced, but… “That’s a bit of an anticlimax,” Harry said. “You’d think there’d be some kind of more dramatic mental event associated with updating on an observation of infinitesimal probability—” Harry stopped himself. Mum, the witch, and even his Dad were giving him \emph{that look} again. “I mean, with finding out that everything I believe is false.”
```
Comment (rrthomas) [inline chapters/hpmor-chapter-002.tex:32]: I think this was correct without the space: it's not the end of a sentence (which would require a full stop after the ellipsis), even though the speech starts with a new sentence.

File: chapters/hpmor-chapter-005.tex [hunk 15]
```diff
@@ -123,7 +123,7 @@ \chapter{The Fundamental Attribution Error}
 
 “And \emph{I} am pleased to learn that I shall be treated with the respect due to the family of Malfoy,” the other boy lobbed back, accompanied by a smile such as the highest of kings might bestow upon the least of his subjects, if that subject were honest, though poor.
 
-Eh…Damn, Harry was having trouble thinking up his next line. Well, everyone \emph{did} want to shake the hand of Harry Potter, so—“When my clothes are fitted, sir, might you deign to shake my hand? I should wish nothing more to put the capper upon this day, nay, this month, indeed, my whole lifetime.”
+Eh…Damn, Harry was having trouble thinking up his next line. Well, everyone \emph{did} want to shake the hand of Harry Potter, so— “When my clothes are fitted, sir, might you deign to shake my hand? I should wish nothing more to put the capper upon this day, nay, this month, indeed, my whole lifetime.”
```
Comment (rrthomas) [inline chapters/hpmor-chapter-005.tex:126]: No space here.

File: chapters/hpmor-chapter-005.tex [hunk 16]
```diff
@@ -137,9 +137,9 @@ \chapter{The Fundamental Attribution Error}
 
 Harry coughed. “Actually, sorry, I’ve got no idea who you are really.”
 
-“\emph{Oh come on!}” the boy said with fierce disappointment. “Why’d you go and do that, then?” Draco’s eyes widened with sudden suspicion. “And how do you \emph{not} know about the Malfoys? And what are those \emph{clothes} you’re wearing? Are your parents \emph{Muggles?}”
+“\emph{Oh come on!}” the boy said with fierce disappointment. “Why’d you go and do that, then?” Draco’s eyes widened with sudden suspicion. “And how do you \emph{not} know about the Malfoys? And what are those \emph{clothes} you’re wearing? Are your parents \emph{Muggles}?”
 
-“Two of my parents are dead,” Harry said. His heart twinged. When he put it that way—“My other two parents are Muggles, and they’re the ones that raised me.”
+“Two of my parents are dead,” Harry said. His heart twinged. When he put it that way— “My other two parents are Muggles, and they’re the ones that raised me.”
```
Comment (rrthomas) [inline chapters/hpmor-chapter-005.tex:142]: No space here.

File: chapters/hpmor-chapter-006.tex [hunk 17]
```diff
@@ -65,7 +65,7 @@ \chapter{The Planning Fallacy}
 
 Professor McGonagall pursed her lips, then shrugged. “I’m still not sure what you mean by ‘scientific experimenting’, Mr~Potter. As I said, I’ve seen Muggle-born students try to get Muggle science to work inside Hogwarts, and people invent new Charms and Potions every year.”
 
-Harry shook his head. “Technology isn’t the same thing as science at all. And trying lots of different ways to do something isn’t the same as experimenting to figure out the rules.” There were plenty of people who’d tried to invent flying machines by trying out lots of things-with-wings, but only the Wright Brothers had built a wind tunnel to measure lift…“Um, how many Muggle-raised children \emph{do} you get at Hogwarts every year?”
+Harry shook his head. “Technology isn’t the same thing as science at all. And trying lots of different ways to do something isn’t the same as experimenting to figure out the rules.” There were plenty of people who’d tried to invent flying machines by trying out lots of things-with-wings, but only the Wright Brothers had built a wind tunnel to measure lift… “Um, how many Muggle-raised children \emph{do} you get at Hogwarts every year?”
```
Comment (rrthomas) [inline chapters/hpmor-chapter-006.tex:68]: No space here.

File: chapters/hpmor-chapter-007.tex [hunk 18]
```diff
@@ -539,15 +541,15 @@ \chapter{Reciprocation}
...
 
-“Persuasive,” Harry said, meaning it descriptively rather than normatively. It was a standard pattern: The Fall from Grace, the need to guard what purity remained against contamination, the past sloping upwards and the future sloping only down. And that pattern also had its \emph{counter}…“I have to correct you on one point of fact, though. Your information about the Muggles is a bit out of date. We aren’t exactly scratching at the dirt any more.”
+“Persuasive,” Harry said, meaning it descriptively rather than normatively. It was a standard pattern: The Fall from Grace, the need to guard what purity remained against contamination, the past sloping upwards and the future sloping only down. And that pattern also had its \emph{counter}… “I have to correct you on one point of fact, though. Your information about the Muggles is a bit out of date. We aren’t exactly scratching at the dirt any more.”
```
Comment (rrthomas) [inline chapters/hpmor-chapter-007.tex:544]: No space here

File: chapters/hpmor-chapter-007.tex [hunk 19]
```diff
@@ -583,13 +585,13 @@ \chapter{Reciprocation}
...
 
 Draco looked up from the book to Harry. His face showed dawning understanding. “Wizards can learn to use this power.”
 
-Very carefully, now…the bait is set, now the hook…“If you can learn to think of yourself as a \emph{human} instead of a \emph{wizard} then you can train and refine your powers as a human.”
+Very carefully, now…the bait is set, now the hook… “If you can learn to think of yourself as a \emph{human} instead of a \emph{wizard} then you can train and refine your powers as a human.”
```
Comment (rrthomas) [inline chapters/hpmor-chapter-007.tex:594]: No space here.

File: chapters/hpmor-chapter-008.tex [hunk 20]
```diff
@@ -1,9 +1,9 @@
...
-All these worlds are J.~K.~Rowling's, except Europa. Attempt no fanfics there.
+All these worlds are J.~K.~Rowling’s, except Europa. Attempt no fanfics there.
 
-One alert reviewer asked whether, if Luna is a seer, that means this is going to be an HPDM bottom!Draco mpreg fic. I regret that FFN does not allow me any larger font size in which to say NO. It honestly hadn't occurred to me that Luna might be a \emph{real} seer—I'll have to decide whether to run with that or not—but I think we can all safely assume that if Luna \emph{is} a seer, she said something about “light planting a seed in darkness”, and Xenophilius, as always, interpreted this in rather the wrong way.
+One alert reviewer asked whether, if Luna is a seer, that means this is going to be an HPDM bottom!Draco mpreg fic. I regret that FFN does not allow me any larger font size in which to say NO. It honestly hadn’s occurred to me that Luna might be a \emph{real} seer—I'll have to decide whether to run with that or not—but I think we can all safely assume that if Luna \emph{is} a seer, she said something about “light planting a seed in darkness”, and Xenophilius, as always, interpreted this in rather the wrong way.
```
Comment (rrthomas) [inline chapters/hpmor-chapter-008.tex:6]: "hadn't" has become "hadn’s"!

File: chapters/hpmor-chapter-008.tex [hunk 21]
```diff
@@ -17,7 +17,7 @@ \chapter{Positive Bias}
 
 There was the sound of an inter-train door opening, and then footsteps and an odd slithering sound coming down the hallway of the train. Hermione laid aside \emph{Hogwarts: A History} and stood up and stuck her head outside—just in case someone needed help—and saw a young boy in a wizard’s dress robes, probably first or second year going by his height, and looking quite silly with a scarf wrapped around his head. A small trunk stood on the floor next to him. Even as she saw him, he knocked on the door of another, closed compartment, and he said in a voice only slightly muffled by the scarf, “Excuse me, can I ask a quick question?”
 
-She didn’t hear the answer from inside the compartment, but after the boy opened the door, she did think she heard him say—unless she’d somehow misheard—“Does anyone here know the six quarks or where I can find a first-year girl named Hermione Granger?”
+She didn’t hear the answer from inside the compartment, but after the boy opened the door, she did think she heard him say—unless she’d somehow misheard— “Does anyone here know the six quarks or where I can find a first-year girl named Hermione Granger?”
```
Comment (rrthomas) [inline chapters/hpmor-chapter-008.tex:20]: No space here.

File: chapters/hpmor-chapter-008.tex [hunk 22]
```diff
@@ -299,7 +298,7 @@ \chapter{Positive Bias}
 
 “Hold on,” came the voice of the \emph{other} boy—Harry Potter. “I’m not sure that’s the best way to do it.”
 
-At this Neville looked like he might cry, and Hermione swung around, angered. If Harry Potter was the sort of person who’d abandon a little boy just because he didn’t want to be interrupted…“What? Why \emph{not?}”
+At this Neville looked like he might cry, and Hermione swung around, angered. If Harry Potter was the sort of person who’d abandon a little boy just because he didn’t want to be interrupted… “What? Why \emph{not}?”
```
Comment (rrthomas) [inline chapters/hpmor-chapter-008.tex:301]: No space here.

File: chapters/hpmor-chapter-009.tex [hunk 23]
```diff
@@ -52,21 +52,21 @@
...
+“I mean—” Harry said even more quietly, trying to figure out how to ask whether goblins had evolved from humans, or evolved from a common ancestor of humans like \emph{Homo erectus}, or if goblins had been \emph{made} out of humans somehow—if, say, they were still genetically human under a heritable enchantment whose magical effect was diluted if only one parent was a ‘goblin’, which would explain how interbreeding was possible, and in which case goblins would \emph{not} be an incredibly valuable second data point for how intelligence had evolved in other species besides \emph{Homo sapiens}—now that Harry thought about it, the goblins in Gringotts \emph{hadn’t} seemed very much like genuinely alien, non-human intelligences, nothing like Dirdir or Puppeteers— “I mean, where did goblins \emph{come} from, anyway?”
```
Comment (rrthomas) [inline chapters/hpmor-chapter-009.tex:69]: No space here.

File: chapters/hpmor-chapter-010.tex [hunk 24]
```diff
@@ -15,11 +15,11 @@
...
+There was a wordless telepathic sigh. “\emph{Though I contain a substantial amount of memory and a small amount of independent processing power, my primary intelligence comes from borrowing the cognitive capacities of the children on whose heads I rest. I am in essence a sort of mirror by which children Sort \emph{themselves}. But most children simply take for granted that a Hat is talking to them and do not wonder about how the Hat \emph{itself} works, so that the mirror is not \emph{self}-reflective. And in \emph{particular} they are not explicitly wondering whether I am fully conscious in the sense of being aware of my own awareness.}”
 
 There was a pause while Harry absorbed all this.
 
-\emph{Oops.}
+\emph{Oops}.
```
Comment (rrthomas) [inline chapters/hpmor-chapter-010.tex:22]: I don't think this needs fixing, the entire sentence can be in `\emph{}`.

File: chapters/hpmor-chapter-012.tex [hunk 25]
```diff
@@ -1,10 +1,10 @@
 \chapter{Impulse Control}
 
 \begin{chapterOpeningAuthorNote}
-ph'nglui mglw'nafh J.~K.~Rowling wgah'nagl fhtagn
+ph‘nglui mglw’nafh J.~K.~Rowling wgah'nagl fhtagn
```
Comment (rrthomas) [inline chapters/hpmor-chapter-012.tex:4]: Quote "wgah'nagl" needs curling.

File: chapters/hpmor-chapter-013.tex [hunk 26]
```diff
@@ -3,10 +3,10 @@ \chapter{Asking the Wrong Questions}
 \begin{chapterOpeningAuthorNote}
 Elen sila J.~K.~Rowling omentielvo.
 
-EDIT: Don't panic. I solemnly swear that there is a logical, foreshadowed, canon-compliant explanation for everything which happens in this chapter. It's a puzzle, you're supposed to try to solve it, and if not, just read the next chapter.
+EDIT: Don’s panic. I solemnly swear that there is a logical, foreshadowed, canon-compliant explanation for everything which happens in this chapter. It’s a puzzle, you're supposed to try to solve it, and if not, just read the next chapter.
```
Comment (rrthomas) [inline chapters/hpmor-chapter-013.tex:6]: "Don't" has become "Don’s".

File: chapters/hpmor-chapter-013.tex [hunk 27]
```diff
@@ -173,7 +196,8 @@ \chapter{Asking the Wrong Questions}
 Opportunities to do good are everywhere\\
 but darkness is where the light needs to be}
 \hbox{\scshape Cost of question: }&\hbox{\scshape 1 point}\\
-\hbox{\scshape Current points: }&\hbox{74}\\[1.5ex]
+\hbox{\scshape Current points: }&\hbox{74}\\%
```
Comment (rrthomas) [inline chapters/hpmor-chapter-013.tex:199]: I don't understand this change.

File: chapters/hpmor-chapter-014.tex [hunk 28]
```diff
@@ -58,13 +58,13 @@ \chapter{The Unknown and the Unknowable}
 
 “I think,” Professor McGonagall said in a rather choked voice, “that you \emph{far} underestimate the rarity of common sense, Mr~Potter.”
 
-That sounded about right. Although…“A Hufflepuff would’ve said the same thing.”
+That sounded about right. Although… “A Hufflepuff would’ve said the same thing.”
```
Comment (rrthomas) [inline chapters/hpmor-chapter-014.tex:61]: No space here.

File: chapters/hpmor-chapter-015.tex [hunk 29]
```diff
@@ -3,10 +3,10 @@ \chapter{Conscientiousness}
 \begin{chapterOpeningAuthorNote}
 Love as thou Rowling.
 
-Today's historical tidbit: The ancient Hebrews considered the boundary of a day to be sunset rather than dawn, so they said “evening and morning” not “morning and evening”. (And as many reviewers noted, modern Jewish halacha asserts the same.)
+Today’s historical tidbit: The ancient Hebrews considered the boundary of a day to be sunset rather than dawn, so they said “evening and morning” not “morning and evening”. (And as many reviewers noted, modern Jewish halacha asserts the same.)
 \end{chapterOpeningAuthorNote}
 \begin{chapterOpeningQuote}
-I'm sure I'll find the time somewhere.
+I‘m sure I’ll find the time somewhere.
```
Comment (rrthomas) [inline chapters/hpmor-chapter-015.tex:9]: Quote in "I'm" curls the wrong way.

File: chapters/hpmor-chapter-015.tex [hunk 30]
```diff
@@ -20,19 +20,19 @@ \chapter{Conscientiousness}
...
 
-Harry had expected to be, in the worst case, second behind Hermione. Harry would have preferred for \emph{her} to be rivalling \emph{him,} of course, but he could have accepted it the other way around.
+Harry had expected to be, in the worst case, second behind Hermione. Harry would have preferred for \emph{her} to be rivalling \emph{him}, of course, but he could have accepted it the other way around.
 
 As of Monday, Harry was headed for the bottom of the class, a position for which he was companionably rivalling all the other Muggle-raised students except Hermione. Who was all alone and rival-less at the top, poor thing.
 
 Professor Flitwick was standing over the desk of one of the other Muggle-borns and quietly adjusting the way she was holding her wand.
 
-Harry looked over at Hermione. He swallowed hard. It was the obvious role for her in the scheme of things…“Hermione?” Harry said tentatively. “Do you have any idea what I might be doing wrong?”
+Harry looked over at Hermione. He swallowed hard. It was the obvious role for her in the scheme of things… “Hermione?” Harry said tentatively. “Do you have any idea what I might be doing wrong?”
```
Comment (rrthomas) [inline chapters/hpmor-chapter-015.tex:35]: No space needed here.

File: chapters/hpmor-chapter-017.tex [hunk 31]
```diff
@@ -170,7 +170,7 @@ \chapter{Locating the Hypothesis}
 
 “Ah,” said Draco, “didn’t \emph{think} of it, I guess that’s why it’s better to have one Ravenclaw as a friend than all of Hufflepuff.”
 
-Oh, hell, how was Harry supposed to juggle this one…“You’re not helping,” Harry said in a mild tone. Hoping Draco would interpret that as \emph{you’re interfering with my plans, please shut up.}
+Oh, hell, how was Harry supposed to juggle this one… “You’re not helping,” Harry said in a mild tone. Hoping Draco would interpret that as \emph{you’re interfering with my plans, please shut up.}
```
Comment (rrthomas) [inline chapters/hpmor-chapter-017.tex:173]: No space needed here.

File: chapters/hpmor-chapter-017.tex [hunk 32]
```diff
@@ -578,19 +578,19 @@ \chapter{Locating the Hypothesis}
...
+Dumbledore started laughing. Laughing a lot harder than Harry would expect, almost howling. It seemed positively \emph{undignified}. An ancient and powerful wizard ought to chuckle in deep booming tones, not laugh so hard he was gasping for breath. Harry had once literally fallen out of his chair while watching the Marx Brothers film \emph{Duck Soup,} and that was how hard Dumbledore was laughing now.
 
 “It’s not \emph{that} funny,” Harry said after a while. He was starting to worry about Dumbledore’s sanity again.
 
 Dumbledore got himself under control again with a visible effort. “Ah, Harry, one symptom of the disease called wisdom is that you begin laughing at things that no-one else thinks is funny, because when you’re wise, Harry, you start getting the jokes!” The old wizard wiped tears away from his eyes. “Ah, me. Ah, me. Oft evil will shall evil mar indeed, in very deed.”
 
-Harry’s brain took a moment to place the familiar words…“Hey, that’s a \emph{Tolkien} quote! \emph{Gandalf} says that!”
+Harry’s brain took a moment to place the familiar words… “Hey, that’s a \emph{Tolkien} quote! \emph{Gandalf} says that!”
```
Comment (rrthomas) [inline chapters/hpmor-chapter-017.tex:589]: No space needed here.

File: chapters/hpmor-chapter-017.tex [hunk 33]
```diff
@@ -612,12 +612,11 @@ \chapter{Locating the Hypothesis}
 
 “It is not,” said Dumbledore, shaking his head and looking very serious. “I took this from the ruins of James and Lily’s home in Godric’s Hollow, where also I found you; and I have kept it from then until now, against the day when I could give it to you.”
 
-In the mixture of hypotheses that served as Harry’s model of the world, Dumbledore’s insanity was rapidly rising in probability. But there \emph{was} still a substantial amount of probability allocated to other alternatives…“Um, is it a \emph{magical} rock?”
+In the mixture of hypotheses that served as Harry’s model of the world, Dumbledore’s insanity was rapidly rising in probability. But there \emph{was} still a substantial amount of probability allocated to other alternatives… “Um, is it a \emph{magical} rock?”
```
Comment (rrthomas) [inline chapters/hpmor-chapter-017.tex:615]: No space needed here.

Comment (entorb) [reply chapters/hpmor-chapter-017.tex:615]: Really? I find it quite dense without, but I am no expert on typesetting...

Comment (rrthomas) [reply chapters/hpmor-chapter-017.tex:615]: I think so! I agree about the look to an extent, but I think it's logical. (If it really needed fixing I'd rather just space the ellipsis a little more.)

Comment (entorb) [reply chapters/hpmor-chapter-017.tex:615]: Ok, than I am gonna revert these changes.

File: chapters/hpmor-chapter-013.tex (hunk 27 above)
Comment (entorb) [reply chapters/hpmor-chapter-013.tex:199]: my bad, I had a script add linebreaks after \\
fixed in https://github.com/rrthomas/hpmor/pull/83

File: chapters/hpmor-chapter-000.tex (hunk 12 above)
Comment (entorb) [reply chapters/hpmor-chapter-000.tex:37]: thanks, fixed as well in https://github.com/rrthomas/hpmor/pull/83

File: chapters/hpmor-chapter-012.tex (hunk 25 above)
Comment (entorb) [reply chapters/hpmor-chapter-012.tex:4]: fixed in https://github.com/rrthomas/hpmor/pull/85/

**Pull request closed without merging**